shardingsphere-scaling/shardingsphere-scaling-core/src/main/java/org/apache/shardingsphere/scaling/core/metadata/MetaDataManager.java

```diff
@@ -43,9 +43,20 @@
      * @return table meta data
      */
     public TableMetaData getTableMetaData(final String tableName) {
+        return getTableMetaData(tableName, "");
```

Comment:
Hi, if you add `getTableMetaData(final String tableName, final String databaseType)` function. This function do not have to exist. and you should update all quote.

How do you know which function can use in which case?

And `databaseType` only affect `Oracle` database, but we don't support `Oracle` in scaling.

So, I suggest keep current code.

Reply:
I understand

Reply:
my origin though is replace `getTableMetaData(final String tableName) ` with `getTableMetaData(final String tableName, final String databaseType) ` one by one , and drop `getTableMetaData(final String tableName) ` at last . since we don't support Oracle . enn , I see